manifest.yml

```diff
@@ -9,6 +9,7 @@ applications:
     FEDERALIST_S3_BUCKET_URL: http://((shared-bucket)).s3-website-us-gov-west-1.amazonaws.com
     DEDICATED_S3_BUCKET_URL: http://$name.s3-website-us-gov-west-1.amazonaws.com
     DNS_RESOLVER: '8.8.8.8'
-    CLOUD_GOV_HOST: cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov
+    # cloud.gov|code.gov|presidentialinnovationfellows.gov|sbst.gov|vote.gov|connect.gov
+    INCLUDE_SUBDOMAINS: cg-88d42ca6-59d7-47e0-9500-4dd9251360b9|federalist-88023b0b-4406-431f-932d-5fd82c70c515|federalist-f7cff978-bf3d-4aaa-bbff-a77bbfc6d827|federalist-c1c3cd8c-125e-41f5-a533-f42e03c01486|cg-9e8debaf-b030-4825-a43c-cb2bc850c96c|federalist-a2074193-6730-4022-a8fd-466df53b6fa5
```

Comment:
Is it worth the complexity of passing this in as an environment variable, rather than listing in the nginx config explicitly?

Reply:
I think the original idea was that the bucket names while not secret, are sort of private so we could put them in an env var so as not to expose them. However, we then put it in the manifest in an OS repo, so that went out the window.

If this were a permanent feature, then I would probably use the origin instead of the host and maybe use a file instead of a regex, but this works as is and we are planning on replacing this proxy application within ~6mos so I don't think it's worth the effort to improve this.